src/huak/lib.rs

```diff
@@ -119,6 +119,8 @@
 //!     └── utils     # Library utilities
 //! ```
 
+extern crate core;
```

Comment:
did this creep in here?

Reply:
huh, I'm not sure where that came from, or the implications. So to answer your question, yes.

Reply:
I can yank it out for you if you want me to. Just give me the 🟢

Reply:
go for it, I need to read up on libcore. and thank you